analyzers/rspec/cs/S3363.html

```diff
@@ -0,0 +1,72 @@
+<p>You should only set a property of a temporal type (like <code>DateTime</code> or <code>DateTimeOffset</code>) as the primary key of a table if the
+date values are guaranteed to be unique. The rule raises an issue if: * Entity Framework, or Entity Framework Core dependencies are found; * a class
```

Comment:
There is something wrong with the adoc format, there should be bullet points here and not stars.